app/components/notice_component.scss

```diff
@@ -0,0 +1,12 @@
+.page-notice {
```

Comment:
Actually, I'm now wondering whether we should scope the css at all here, e.g. .component-page-notice, or .notice-component?